<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="mrogers40-PC" name="test.SolitaireLayoutTest" tests="6" failures="0" timestamp="30 Apr 2017 07:28:14 GMT" time="0.015" errors="2">
  <testcase name="minimumLayoutSize" time="0.001" classname="test.SolitaireLayoutTest">
    <error type="org.testng.TestException" message="
Method SolitaireLayoutTest.minimumLayoutSize()[pri:0, instance:test.SolitaireLayoutTest@89484] should have thrown an exception of type class java.lang.UnsupportedOperationException">
      <![CDATA[org.testng.TestException: 
Method SolitaireLayoutTest.minimumLayoutSize()[pri:0, instance:test.SolitaireLayoutTest@89484] should have thrown an exception of type class java.lang.UnsupportedOperationException
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:89)
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1416)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:695)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- minimumLayoutSize -->
  <testcase name="removeLayoutComponent" time="0.002" classname="test.SolitaireLayoutTest"/>
  <testcase name="layoutContainer" time="0.005" classname="test.SolitaireLayoutTest"/>
  <testcase name="addLayoutComponent" time="0.003" classname="test.SolitaireLayoutTest"/>
  <testcase name="preferredLayoutSize" time="0.002" classname="test.SolitaireLayoutTest">
    <error type="org.testng.TestException" message="
Method SolitaireLayoutTest.preferredLayoutSize()[pri:0, instance:test.SolitaireLayoutTest@89484] should have thrown an exception of type class java.lang.UnsupportedOperationException">
      <![CDATA[org.testng.TestException: 
Method SolitaireLayoutTest.preferredLayoutSize()[pri:0, instance:test.SolitaireLayoutTest@89484] should have thrown an exception of type class java.lang.UnsupportedOperationException
	at org.testng.internal.ExpectedExceptionsHolder.noException(ExpectedExceptionsHolder.java:89)
	at org.testng.internal.Invoker.handleInvocationResults(Invoker.java:1416)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:695)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- preferredLayoutSize -->
  <testcase name="SolitaireLayout" time="0.002" classname="test.SolitaireLayoutTest"/>
</testsuite> <!-- test.SolitaireLayoutTest -->
